imports/plugins/included/connectors-shopify/package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file should almost certainly not be included as I don't think we're including any package-lock.json files currently

Reply:
Oops!!!